### PR TITLE
Convert responsive element to CSS modules

### LINKED ISSUE #561: Convert Responsive Element component to CSS modules
# Issue Description
To better align with modern CSS practices and modern CSS theming capabilities, we want to update this component to use CSS modules and if applicable, CSS custom properties.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Component styles are converted to use CSS modules. If applicable, convert SCSS variables to CSS custom properties.

## Current Behavior
Component styles use manual name spaced classes and SCSS variables.

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Resolves #561 

### Deployment Link
https://terra-core-deployed-pr-761.herokuapp.com/static/#/site/responsive-element

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

File: packages/terra-modal-manager/src/ModalManager.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Modal from 'terra-modal';
 
 import AppDelegate from 'terra-app-delegate';
 import SlideGroup from 'terra-slide-group';
-import getBreakpoints from 'terra-responsive-element/lib/breakpoints';
+import breakpoints from 'terra-responsive-element/lib/breakpoints.scss';
```
Comment (mjhenkes) [inline packages/terra-modal-manager/src/ModalManager.jsx]: Does this make breakpoints not themeable?

Comment (mhemesath) [reply packages/terra-modal-manager/src/ModalManager.jsx]: They can be themed at build time

Comment (mhemesath) [reply packages/terra-modal-manager/src/ModalManager.jsx]: They can be themed at build time

Comment (mhemesath) [reply packages/terra-modal-manager/src/ModalManager.jsx]: They can be themed at build time

Comment (bjankord) [reply packages/terra-modal-manager/src/ModalManager.jsx]: Correct, we can pass in a file to override variables or override them directly similar to how we set themes in the sass-loader.

Comment (mhemesath) [reply packages/terra-modal-manager/src/ModalManager.jsx]: Media queries can't actually be themed with CSS custom properties anyway.

**Comment by mmalaker** [comment]
Functional verification completed. The responsive element examples display and function correctly after conversion to CSS modules.

**Pull request merged**